gradle/common/java11Options.gradle

```diff
@@ -23,12 +23,22 @@ project.ext.JDK11_OPTIONS = ['-Djdk.attach.allowAttachSelf=true',
                              '--add-exports', 'java.management.rmi/com.sun.jmx.remote.internal.rmi=ALL-UNNAMED',
                              '--add-exports', 'java.rmi/sun.rmi.registry=ALL-UNNAMED',
                              '--add-exports', 'java.rmi/sun.rmi.server=ALL-UNNAMED',
+                             '--add-exports', 'java.rmi/sun.rmi.transport=ALL-UNNAMED',
```

Comment:
it'd be nice to document which flags are needed for JDK 11 and which ones are needed for JDK 17